components/widgets/land-cover/tree-cover-density/index.js

```diff
@@ -0,0 +1,67 @@
+import { getTreeCoverDensity } from 'services/analysis-cached';
+
+import {
+  POLITICAL_BOUNDARIES_DATASET,
+  TROPICAL_TREE_COVER,
+} from 'data/datasets';
+import {
+  DISPUTED_POLITICAL_BOUNDARIES,
+  POLITICAL_BOUNDARIES,
+  TROPICAL_TREE_COVER_METERS,
+  TROPICAL_TREE_COVER_HECTARE,
+} from 'data/layers';
+
+import getWidgetProps from './selectors';
+
+export default {
+  widget: 'treeCoverDensity',
+  title: 'Tree cover density in {location}',
+  categories: ['land-cover'],
+  types: ['country', 'wdpa', 'aoi'],
+  admins: ['adm0', 'adm1', 'adm2'],
+  large: true,
+  visible: ['dashboard', 'analysis'],
+  chartType: 'composedChart',
+  colors: 'density',
+  settingsConfig: [
+    {
+      key: 'landCategory',
+      label: 'Land Category',
+      type: 'select',
+      placeholder: 'All categories',
+      clearable: true,
+      border: true,
+    },
+  ],
+  pendingKeys: ['threshold', 'years'],
+  refetchKeys: ['landCategory'],
+  dataType: 'gain',
+  // TO-DO: Add metadata URL
+  metaKey: 'widget_primary_forest',
+  datasets: [
+    {
+      dataset: POLITICAL_BOUNDARIES_DATASET,
+      layers: [DISPUTED_POLITICAL_BOUNDARIES, POLITICAL_BOUNDARIES],
+      boundary: true,
+    },
+    {
+      dataset: TROPICAL_TREE_COVER,
+      layers: [TROPICAL_TREE_COVER_METERS, TROPICAL_TREE_COVER_HECTARE],
```

Comment:
I might be reading this wrong, but I think you're defining the map to show both layers (meters and hectare) when the user clicks the Map button. As per Teresa's last comment, it should show the 0.5ha layer, or `TROPICAL_TREE_COVER_HECTARE`. I think you need to remove `TROPICAL_TREE_COVER_METERS`.